src/content/release-notes.mdx

```diff
@@ -5,6 +5,43 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.25.0
+
+7 October 2024
+
+This versions is compatible with Kubernetes versions 1.27 to 1.30
+
+### New Features
+
+- [Announcing Okteto CLI 3.0](https://www.okteto.com/blog/cli-three-release/) - upgrading to chart release 1.25 requires a CLI upgrade to 3.0
+- Announcing Cloud Credentials, a central location to manage your authentication with AWS and GCP  <!-- 8550 -->
+- Enabling Okteto Insights for all users of Okteto <!-- 8627 -->
+- Added configurable buildkit network mode  <!-- 8486 -->
+- Init dynamic config  <!-- https://github.com/okteto/app/pull/8475 -->
+- Added CLI, Chart, and Okteto App version to our API headers  <!-- 8545 -->
+- Support to enable/disable resource quotas for individual Namespaces  <!-- 8604 -->
+- Allow override of installer security context  <!-- https://github.com/okteto/app/pull/8608 -->
+
+### Improvements
+
+- Show more actionable feedback upon "failed to deploy okteto pipeline"…  <!-- 8521 -->
+- You can now list your Github App installations under Settings → Integrations  <!-- 8463 -->
+- Implemented a number of UI updates to ensure your Okteto experience stays smooth <!-- 8600, 8623, 8534, 8626 -->
+
+### Bug Fixes
```

Comment:
There is a relevant fix in the CLI that it worths to be mentioned. It is a fix for Smart Builds. It would be something like:

`Fix in Smart Builds logic to properly calculate when to build a new image if the build context points to a parent folder`